plugins/Denizen/scripts/utility/player_restraining.dsc

```diff
@@ -0,0 +1,14 @@
+player_restraints:
+    type: world
+    debug: false
+    events:
+        on player walks flagged:util.nowalk:
+            - determine passively cancelled
+        on player steps on block flagged:util.nostep:
```

Comment:
wouldn't this do the same thing as the event above?

Reply:
no walks is completely different than steps on

Reply:
oh